### ISSUE TITLE
Function keys F1/F2 stopped working inside of tmux 3.4

### ISSUE THREAD

**Issue by casperklein** [tmux/tmux#3876]
### Issue description

I use the following commands in bash, to be able to use the F1 / F2 keys to start `uptime` and `htop`:

```bash
bind '"\e[11~":"uptime\n"'
bind '"\e[12~":"htop\n"'
```

That worked fine inside tmux 3.3a, but started to fail with version 3.4.

After some testing I noticed, only F1-F4 is affected. If I use F5/F6 instead, it works.

### Required information

Please provide the following information:

* tmux version: tmux 3.4
* Platform: Linux unknown
* $TERM inside of tmux: tmux-256color
* $TERM outside of tmux: xterm-256color
* Logs from tmux (`tmux kill-server; tmux -vv new`):
  * [tmux-client-2457.log](https://github.com/tmux/tmux/files/14546018/tmux-client-2457.log)
  * [tmux-out-2459.log](https://github.com/tmux/tmux/files/14546019/tmux-out-2459.log)
  * [tmux-server-2459.log](https://github.com/tmux/tmux/files/14546020/tmux-server-2459.log)

**Comment by amitzionii** [comment]
Unrelated directly to tmux 
it seems that the right keycodes for you bind command should be:

``
bind '"\eOP":"uptime\n"'
bind '"\eOQ":"htop\n"'
``
Could you please try that ?

**Comment by nicm** [comment]
`\E[11~` is rxvt's F1 sequence, before 3.3 tmux did not have a builtin mapping for this and relied on `TERM` (which in your case is wrong).

tmux will send `\E[OP` for F1 like xterm.

**Comment by casperklein** [comment]
I got it fixed. I am using PuTTY (sorry, I didn't mention that before).

PuTTY sets TERM=xterm, but uses the wrong? key sequences `[11~` in it's default configuration.

Fix: In the configuration of PuTTY under "Terminal - Keyboard" I switched the `The function keys and keypad` setting from `ESC[n~` to `Xterm R6`.

After correcting my bash bindings from `\e[11~` to `\eOP` etc., the function keys now work inside and outside of tmux.
